pkg/cmd/kafka/update/update.go

```diff
@@ -153,15 +159,40 @@ func run(opts *options) error {
 		if err != nil {
 			return err
 		}
-	}
 
-	if opts.owner == kafkaInstance.GetOwner() {
-		opts.logger.Info(opts.localizer.MustLocalize("kafka.update.log.info.sameOwnerNoChanges", localize.NewEntry("Owner", opts.owner), localize.NewEntry("InstanceName", kafkaInstance.GetName())))
-		return nil
+		enableReauthPrompt := &survey.Select{
+			Message: opts.localizer.MustLocalize("kafka.update.input.message.enableReauth"),
+			Options: []string{"true", "false"},
+			Default: strconv.FormatBool(kafkaInstance.GetReauthenticationEnabled()),
+		}
+
+		err = survey.AskOne(enableReauthPrompt, &opts.enableReauth)
+		if err != nil {
+			return err
+		}
 	}
 
 	updateObj := kafkamgmtclient.NewKafkaUpdateRequest()
-	updateObj.SetOwner(opts.owner)
+
+	if opts.owner != "" && opts.owner != kafkaInstance.GetOwner() {
```

Comment:
This logic missing else and `"kafka.update.log.info.sameOwnerNoChanges` that were there previously. 
When adding flags we can eather:

- Make command execute only single operation per flag (owner, reauth)
- Make command execute multiple actions at the same time - will require priorities and decision how command should fail - Aggregation of errors that we done for acl etc.

Reply:
> will require priorities and decision how command should fail - Aggregation of errors that we done for acl etc.
I think this is something backend takes care of already, running a `kafka update --owner inexistent_user --reauthentication false` fails with error `user doesn't belong to your organization` and doesn't update the enable_reauthtentication field

Reply:
Yes, but what is the plan to handle this in the CLI?

Do we want to even send request if we know that it will be wrong?
Do we provide info to user that error in owner also discarded other update (that might be assumed, but better to be explicit)

Reply:
> Do we want to even send request if we know that it will be wrong?

While we can check by fetching the list of users before sending request for update. I think we should let the backend check it for us.

> Do we provide info to user that error in owner also discarded other update

I would prefer not to be explicit about the discard as update sounds very much atomic. I will let you take the final call on it, maybe we could display a simple message like `update aborted`.

Reply:
Please use your jugement based on my review. I just point out that relying on server side (especially server doesn't provide reliable error codes for that case that will help us to provide proper message to user) is bad UX. 

See entire terms and conditions check - backend does that but we still check that before :)

Reply:
current preview:
![Screenshot from 2021-12-02 13-20-22](https://user-images.githubusercontent.com/23582438/144380005-0bfa62b2-fbaf-46f0-b47c-c70eb85d1dab.png)